### ISSUE TITLE
 Error while Loading

### ISSUE THREAD

**Issue by Raining-Cloud** [bw2012/UE4VoxelTerrain#30]
I cloned the Project multiple times but every Time I have Issues with the Version. I use UE 4 24.3

**Comment by bw2012** [comment]
Hi. I have tested it again and it works for me with current github version on UE 4.24.3
Could you please provide log files or screenshot of error?

**Comment by Raining-Cloud** [comment]
![grafik](https://user-images.githubusercontent.com/59502349/79437140-12978100-7fd2-11ea-9982-7bb962558ff2.png)
This is the Error. I have the Epic Launcher UE Version does this make any difference?

**Comment by bw2012** [comment]
Could you please provide log file? 
UE4VoxelTerrain\UE4VoxelTerrain\Saved\Logs\UE4VoxelTerrain.log

**Comment by Raining-Cloud** [comment]
[UE4VoxelTerrain.log](https://github.com/bw2012/UE4VoxelTerrain/files/4486791/UE4VoxelTerrain.log)

**Comment by bw2012** [comment]
Thanks. It looks like VS2019 specific error. I will check it asap

**Comment by Raining-Cloud** [comment]
Ok because I wanted to make dynamic snow with this. Do you think I should download VS17 too?

**Comment by lolaswift** [comment]
@bw2012 hi still having the issue.

**Comment by bw2012** [comment]
@lolaswift with VS2019?

**Comment by lolaswift** [comment]
@bw2012 Yes. VS2019. Multiple errors. For example, "Severity	Code	Description	Project	File	Line	Suppression State
Error		Could not find definition for module 'UnrealSandboxTerrain', (referenced via Target -> UE4VoxelTerrain.Build.cs)	UE4VoxelTerrain	D:\UE4VoxelTerrain-master\UE4VoxelTerrain\Intermediate\ProjectFiles\UnrealBuildTool	1	
"

**Comment by bw2012** [comment]
@lolaswift I have tested it with vs2019 and it works for me. Could you please attach log file? UE4VoxelTerrain\UE4VoxelTerrain\Saved\Logs\UE4VoxelTerrain.log

**Comment by Webkadabra** [comment]
this project is awesome, and everythign works fine, people! This is gold, actually, for me as a hobbyst Unreal Engine student. Just wanted to say this, since there seems to be way too many issues created here with qustions like "how do I code, please?" lol. Thank you for sharing, this project is awesome!

PS
I run 64 gig of ram. Maybe if you have 8-16 gigs, no point in getting into learning industry top 3D games engine? just sayin...
